code/modules/cargo/bounties/assistant.dm

```diff
@@ -76,44 +70,45 @@
 	reward = 1250
 	wanted_types = list(/obj/item/extendohand)
 
-/datum/bounty/item/assistant/donut
-	name = "Donuts"
-	description = "CentCom's security forces are facing heavy losses against the Syndicate. Ship donuts to raise morale."
-	reward = 2000
-	required_count = 10
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/donut)
+// /datum/bounty/item/assistant/donut
+// 	name = "Donuts"
+// 	description = "CentCom's security forces are facing heavy losses against the Syndicate. Ship donuts to raise morale."
+// 	reward = 2000
+// 	required_count = 10
+// 	wanted_types = list(/obj/item/reagent_containers/food/snacks/donut)
 
-/datum/bounty/item/assistant/donkpocket
-	name = "Donk-Pockets"
-	description = "Consumer safety recall: Warning. Donk-Pockets manufactured in the past year contain hazardous lizard biomatter. Return units to CentCom immediately."
-	reward = 1000
-	required_count = 10
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/donkpocket)
+// /datum/bounty/item/assistant/donkpocket
+// 	name = "Donk-Pockets"
+// 	description = "Consumer safety recall: Warning. Donk-Pockets manufactured in the past year contain hazardous lizard biomatter. Return units to CentCom immediately."
+// 	reward = 1000
+// 	required_count = 10
+// 	wanted_types = list(/obj/item/reagent_containers/food/snacks/donkpocket)
 
 /datum/bounty/item/assistant/briefcase
 	name = "Briefcase"
 	description = "Central Command will be holding a business convention this year. Ship a few briefcases in support."
-	reward = 1500
-	required_count = 5
+	reward = 1200
+	required_count = 3
 	wanted_types = list(/obj/item/storage/briefcase, /obj/item/storage/secure/briefcase)
 
 /datum/bounty/item/assistant/sunglasses
 	name = "Sunglasses"
-	description = "A famous blues duo is passing through the sector, but they've lost their shades and they can't perform. Ship new sunglasses to CentCom to rectify this."
-	reward = 1000
-	required_count = 2
+	description = "A famous bounty hunter is passing through the sector, and they've lost their shades. They're too embarrassed to hunt criminals with their naked eyes. Ship new sunglasses to CentCom to rectify this."
+	reward = 1750
```

Comment:
Lowering this to 1000 is a good idea, its just one pare of glasses and they are easy to come by. I.e you can spawn with them...

Reply:
Only certain jobs spawn with sunglasses, and they're a fairly powerful item to give up for a bounty. (Flash Immunity with no vision impairment) My concern was mostly reducing the count, as getting sunglasses as an assistant can be a pain.

Reply:
You spawn in loadout any job with sunglasses

Reply:
Unless you have some sort of admin or donor access that I don't access to, the only job that can spawn with "sunglasses" in the loadout are security, by spawning in with sec-huds. Otherwise, as far as I know, the QM and Bartender spawn with either sunglasses, or items that can be made into sunglasses.